File: Iniciante/URI 2930.hs
main :: IO ()
main = do
    line <- getLine
    let (e:d:_) = map (read :: String -> Double) $ words line
    if e > d
        then putStrLn $ "Eu odeio a professora!"
    else if (d - e) >= 3
        then putStrLn $ "Muito bem! Apresenta antes do Natal!"
    else if (e + 2) < 24
        then putStrLn $ "Parece o trabalho do meu filho!\nTCC Apresentado!"
    else
        putStrLn $ "Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!"

File: Iniciante/URI 2930.rb
line = gets.split(' ')
e = line[0].to_i
d = line[1].to_i

if e > d
    puts "Eu odeio a professora!"
elsif (d - e) >= 3
    puts "Muito bem! Apresenta antes do Natal!"
elsif (e + 2) < 24
    puts "Parece o trabalho do meu filho!\nTCC Apresentado!"
else
    puts "Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!"
end